Bottom: 2a851356dedaf0d611560fc342b239ed48b73bbd
Top:    420aa00e48a82369b66be34d4633b4966f3ff029
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-01-09 18:41:33 +0530

Refresh of sched-core-add-manual

---

diff --git a/kernel/sched/sched.h b/kernel/sched/sched.h
index 71e2b19de51b..cf7f40a06d4e 100644
--- a/kernel/sched/sched.h
+++ b/kernel/sched/sched.h
@@ -2499,5 +2499,5 @@ static inline void membarrier_switch_mm(struct rq *rq,
 #endif
 
 #define lenient_latency(lat)	((lat) == MAX_LATENCY_TOLERANCE)
-#define task_latency_lenient(lat)	\
+#define task_latency_lenient(p)	\
 	(lenient_latency((p)->latency_tolerance))
